.github/workflows/bench.yml

```diff
@@ -6,11 +6,20 @@ on:
       bench-file:
         required: true
         type: string
+      node-versions:
+        required: true
+        type: string
+        default: '["16.0.0", "16.18.1", "16.19.0", "16.20.2", "18.0.0", "18.16.1", "18.17.1", "18.18.0", "19.0.0", "19.9.0", "20.0.0", "20.6.0", "20.7.0", "20.8.0"]'
+        description: 'The Node.js Versions (should be a JSON array)'
   workflow_call:
     inputs:
       bench-file:
         required: true
         type: string
+      node-versions:
+        required: true
```

Comment:
if it has a default it shouldn't be required, right?

Reply:
I think you can still pass empty if you don't have the required.

Reply:
let's try